### PR TITLE
OD 692

### PR DESCRIPTION
**Pull request by SimengSun**
OD 692 Map single original tag to multiple AO3 tags (also OD-574 insert entries in item_tags for the new tags)

Added unit test and test sql data. The first test example is the same one described in the OD-692 ticket.

### CODE REVIEW

File: shared_python/Tags.py [hunk 1]
```diff
@@ -133,11 +132,13 @@ def update_tag_row(self, row: dict):
     ao3_tag_types = row[tag_headers['ao3_tag_type']].split(",")
     number_types = len(ao3_tag_types)
 
+    num_insert = 0
     # If tags length > types length -> there are remapped tags
     # Iterate over all the provided AO3 tags:
     # - First tag -> update the existing row
     # - Other tags -> create new row in tags table
     for idx, ao3_tag in enumerate(ao3_tags):
+      ao3_tag = ao3_tag.lstrip().rstrip()
```
Comment (ariana-paris) [inline shared_python/Tags.py:141]: Looks as if Python's `strip` might do the trick here.

File: shared_python/Tags.py [hunk 2]
```diff
@@ -155,7 +156,18 @@ def update_tag_row(self, row: dict):
...
-        # FIXME OD-574 need to also insert entries in item_tags for the new tags
+        # get last auto increment tag id
+        sql_dict = self.sql.execute_dict(f"""select LAST_INSERT_ID();""")
+        new_tag_id = sql_dict[0]['LAST_INSERT_ID()']
+        # get all associated items from item_tags
+        items = self.sql.execute_dict(f"""SELECT item_id, item_type 
+                                          FROM item_tags WHERE tag_id = {row['Original Tag ID']}""")
+        
+        # insert into item_tags table
+        for item in items:
```
Comment (ariana-paris) [inline shared_python/Tags.py:167]: 🎉  So happy to have this finally done!

**Review by ariana-paris — COMMENTED** [review]
Just one comment but very excited to see this feature implemented!

File: shared_python/Tags.py (hunk 1 above)
Comment (SimengSun) [reply shared_python/Tags.py]: yes it does! just replaced with `strip()`

**Review by ariana-paris — APPROVED** [review]

**Pull request merged**